frontend/src/app/modules/bookings/bookings-page/bookings-page.component.ts

```diff
@@ -80,12 +80,10 @@ export class BookingsPageComponent extends BaseComponent implements OnInit, OnDe
         const containerWidth = this.getPageSize();
 
         this.getNumberOfItemsToDisplay(containerWidth);
-        this.teamService.currentTeamEmitted$
-            .subscribe(teamId => {
-                this.teamId = teamId;
-
-                this.loadMeetings(this.getMeetingRequest());
-            });
+        this.teamService.currentTeamEmitted$.pipe(this.untilThis).subscribe((teamId) => {
+            this.teamId = teamId;
+        });
+        this.loadMeetings(this.getMeetingRequest());
```

Comment:
Use RxJS

Reply:
I will refactor this code. But we can't use RxJS because method getMeetingReques doesn't return Observable  https://prnt.sc/8CftD4oXVAUY